functions/mqhandlers.go

```diff
@@ -141,6 +141,11 @@ func HostPeerUpdate(client mqtt.Client, msg mqtt.Message) {
 	}
 	if peerUpdate.ServerVersion != config.Version {
 		logger.Log(0, "server/client version mismatch server: ", peerUpdate.ServerVersion, " client: ", config.Version)
+		if err := UseVersion(peerUpdate.ServerVersion, false); err != nil {
```

Comment:
This does not handle the case where netclient is newer than netmaker and will result in downgrading netclient

Reply:
done